native/general.go

```diff
@@ -4,22 +4,65 @@
 
 package native
 
-import "github.com/gonum/lapack"
+import (
+	"math"
+
+	"github.com/gonum/lapack"
+)
 
 type Implementation struct{}
 
+type Blah struct {
+	A int
+	B float64
+}
+
 var _ lapack.Float64 = Implementation{}
 
 const (
-	badUplo = "lapack: illegal triangle"
-	nLT0    = "lapack: n < 0"
-	badLdA  = "lapack: index of a out of range"
+	badDirect     = "lapack: bad direct"
+	badLdA        = "lapack: index of a out of range"
+	badSide       = "lapack: bad side"
+	badStore      = "lapack: bad store"
+	badTau        = "lapack: tau has insufficient length"
+	badTrans      = "lapack: bad trans"
+	badUplo       = "lapack: illegal triangle"
+	badWork       = "lapack: insufficient working memory"
+	badWorkStride = "lapack: insufficient working array stride"
+	negDimension  = "lapack: negative matrix dimension"
+	nLT0          = "lapack: n < 0"
+	shortWork     = "lapack: working array shorter than declared"
 )
 
 func blockSize() int {
```

Comment:
Add a comment for this - not exported, but it will be useful to know why 64 is chosen for example.